promptsource/templates/clue/c3/templates.yaml

```diff
@@ -0,0 +1,65 @@
+dataset: clue
+subset: c3
+templates:
+  51b3c3fe-2fa2-474a-81f9-5b421c884109: !Template
+    answer_choices: '{{ choice | join(" ||| ") }}'
+    id: 51b3c3fe-2fa2-474a-81f9-5b421c884109
+    jinja: "{% for statement in context %} \n{{ statement }}\n{% endfor %}\nGiven\
+      \ the dialogue / passage above, use the following options to answer the question\
+      \ \"{{question}}\".\nOptions: \n- {{ answer_choices | join('\\n- ') }}\n|||\n\
+      {{ answer }}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      languages:
+      - zh
+      metrics:
+      - Accuracy
+      original_task: true
+    name: answer-question-affirmative
+    reference: ''
+  5e06f05f-d7dd-4329-b6d8-3a62dcdba838: !Template
+    answer_choices: '{{ choice | join(" ||| ") }}'
+    id: 5e06f05f-d7dd-4329-b6d8-3a62dcdba838
+    jinja: "Question: \"{{question}}\"\nAnswer choices: {{ answer_choices[:-1] | join(',\
+      \ ') }}, or  {{ answer_choices[-1] }}?\nPassage: {% for statement in context\
+      \ %} \n{{ statement }}\n{% endfor %}\n|||\n{{ answer }}"
```

Comment:
This renders as e.g. `['Given the dialogue / passage below, what is the answer for the question "根据对话，可以知道什么?"\nAnswer choices: 今天天气不好, 比赛时间变了, or  校长忘了时间?\n \n男：足球比赛是明天上午八点开始吧?\n \n女：因为天气不好，比赛改到后天下午三点了。', '比赛时间变了']`

How should the model know where the passage actually ends?
It may be reasonable to just continue the previous passage. Fine-tuning on such examples may lead to generation quality decreasing I think cc @thomasw21

Reply:
I don't understand the point you're making. I don't know why the rendering is a list ...

Reply:
My bad the first one is the input, the second the answer. So if we separate them with a whitespace the model will get:

`Given the dialogue / passage below, what is the answer for the question "根据对话，可以知道什么?"\nAnswer choices: 今天天气不好, 比赛时间变了, or 校长忘了时间?\n \n男：足球比赛是明天上午八点开始吧?\n \n女：因为天气不好，比赛改到后天下午三点了。 比赛时间变了`

Reply:
Ah I understand, yeah good point, I think putting the passage above makes more sense
```
{passage} 

Given the dialogue / passage below, what is the answer for the question "根据对话，可以知道什么?"\nAnswer choices: 今天天气不好, 比赛时间变了, or 校长忘了时间?
```

One other way of doing it is have a <eos> between input and target, which we've been avoiding but in this case it might make sense?

Nit: also you can remove the answer choices for him to figure out (much harder task, but would maybe help quite a bit the training?

Reply:
` is have a between ` you mean have a EOS token?

Agreed, let's add another prompt without answer choices if you agree @yongzx?

Reply:
Yes woops

Reply:
Yeap I can do that.

Reply:
@Muennighoff @thomasw21 For prompts without answer choices, should I mark it as non-original because I think we should use ROUGE or other generation metrics instead of original metric "accuracy", as we are no longer choosing answer from the given answer options.

Reply:
I don't know, I'm not too familiar with how that terminology is used for. Perhaps @VictorSanh can help to know this kind of things.